src/cli/onefuzz/templates/libfuzzer.py

```diff
@@ -72,6 +74,12 @@ def _create_tasks(
             ),
         ]
 
+        # We don't really need a separate timeout for crash reporting, and we could just
```

Comment:
Compare with definition of `coverage_timeout` below.

Open to suggestions here; we could also just NOT introduce a new template arg, reuse `crash_report_timeout`, and unify them later as a breaking change.